packages/protocol/contracts/common/Accounts.sol

```diff
@@ -323,15 +323,27 @@ contract Accounts is
    * @param fraction The fraction of the validator's payment that should be
    * diverted to `beneficiary` every epoch, given as FixidyLib value. Must not
    * be greater than 1.
+   * @dev Use `deletePaymentDelegation` to unset the payment delegation.
    */
   function setPaymentDelegation(address beneficiary, uint256 fraction) public {
     require(isAccount(msg.sender), "Not an account");
+    require(beneficiary != address(0), "Beneficiary cannot be address 0x0");
     FixidityLib.Fraction memory f = FixidityLib.wrap(fraction);
     require(f.lte(FixidityLib.fixed1()), "Fraction must not be greater than 1");
     paymentDelegations[msg.sender] = PaymentDelegation(beneficiary, f);
     emit PaymentDelegationSet(beneficiary, fraction);
   }
 
+  /**
+   * @notice Removes a validator's payment delegation by setting benficiary and
+   * fraction to 0.
+   */
+  function deletePaymentDelegation() public {
```

Comment:
nitpick. do not fully understand the logic yet and it seems to me like delete* shouldn't be publicly visible 🤔